packages/hoprd/src/hopr.ts

```diff
@@ -275,32 +117,12 @@ export async function createHoprNode(
     db,
     tagBloomFilter,
     storeTagBloomFilterContent,
-    connector.sendTransaction.bind(connector),
     message_emitter as WasmHoprMessageEmitter,
-    chain_query as WasmChainQuery,
     onReceivedMessage,
     onAcknowledgement
   )
 
-  log('Setting up the indexer events...')
-  let hc = HoprCoreEthereum.getInstance()
-
-  let continueStartup = defer<void>()
-  hc.on('hopr:connector:created', () => {
-    // 2.b - Connector has been created, and we can now trigger the next set of steps.
-    log('Connector has been loaded properly.')
-    continueStartup.resolve()
-  })
-
-  // 2.a - Setup connector listener to bubble up to node. Emit connector creation.
-  log(`Ready to request on-chain connector to connect to provider.`)
-  hc.emit('connector:create')
-
-  await continueStartup.promise
-
   let processes = await hopr.run()
```

Comment:
This is also no longer necessary. Only hopr object can be passed around... and then triggered to do run in the upper calling layer.